### ISSUE TITLE
Publish Error: "Refusing to allow an OAuth App to create or update workflow"

### ISSUE THREAD

**Issue by caleb531** [raycast/extensions#10850]
### Description

Hi,

I was trying to use the `ray publish` command to publish an update to my extension (YouVersion Bible Suggest). I get past the GitHub OTP step, but receive an error from what seems like the GitHub API:

```
error -  getting fork
fork extensions repo: could not get the latest changes. Head to https://github.com/caleb531/raycast-extensions, select the Sync fork dropdown menu above the list of files, and then click Update branch. Once you've done that, try running this command again

Error: Error Body: {"message":"refusing to allow an OAuth App to create or update workflow `.github/workflows/extensions_build_publish.yml` without `workflow` scope","documentation_url":"https://docs.github.com/rest/branches/branches#sync-a-fork-branch-with-the-upstream-repository"}
```

I have checked my own (classic) personal access token, and can confirm that it has the _Workflow_ scope enabled, although I mostly did this as a sanity check—I would think that OAuth apps use their own tokens for authentication.

In the meantime, I have synced my fork and submitted a PR to work around the issue.

### Steps To Reproduce

1. Run `npm run publish` or `npm exec ray publish`
2. Press Enter to open the GitHub authentication page
3. Enter the OTP provided by the Raycast CLI into GitHub
4. Return to the terminal

### Current Behaviour

The following is displayed in the terminal:

```
error -  getting fork
fork extensions repo: could not get the latest changes. Head to https://github.com/caleb531/raycast-extensions, select the Sync fork dropdown menu above the list of files, and then click Update branch. Once you've done that, try running this command again

Error: Error Body: {"message":"refusing to allow an OAuth App to create or update workflow `.github/workflows/extensions_build_publish.yml` without `workflow` scope","documentation_url":"https://docs.github.com/rest/branches/branches#sync-a-fork-branch-with-the-upstream-repository"}
```

### Expected Behaviour

I would expect the publish command to complete successfully by opening a pull request (or whatever the normal behavior is).

**Comment by vineus** [comment]
I had the same issue. Fixed by following what the error message says:

> Head to https://github.com/caleb531/raycast-extensions, select the Sync fork dropdown menu above the list of files, and then click Update branch. Once you've done that, try running this command again

**Comment by mathieudutour** [comment]
Yeah we don't request the workflow scope since it's really powerful and we don't _really_ need it. GitHub doesn't provide a way to only access the workflow when merging things so we need the workaround...
